skbio/sequence/distance.py

```diff
@@ -113,3 +104,54 @@ def hamming(seq1, seq2):
         distance = scipy.spatial.distance.hamming(seq1.values, seq2.values)
 
     return float(distance)
+
+
+@experimental(as_of='0.4.2')
+def kmer_distance(seq1, seq2, k=3, overlap=True):
+    """Compute the kmer distance between a pair of sequences
+    Parameters
+    ----------
+    seq1 : skbio.Sequence
+    seq2 : skbio.Sequence
+    k : int, optional
+        The word length.
+    overlap : bool, optional
+        Defines whether the k-words should be overlapping or not
+        overlapping.
+    Returns
+    -------
+    float
+        Fraction of the set of k-mers from both seq1 and
```

Comment:
This text describes what kmer distance is, so I'd move it above "Parameters" so it's more visible. Here's a reworded version to use there:

*The kmer distance between two sequences is the fraction of kmers that are unique to either sequence.*

Change the text here to be:

*kmer distance between `seq1` and `seq2`.*